### PR TITLE
Layout changes to PR template

### PR DESCRIPTION
**Pull request by alxs**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,19 +1,15 @@
...
+> Please provide a brief description of your project here summarising key points (1-2 paragraphs).
 >
-> The details should be in the application document that is being added to the repository via this pull request.
+> If your application is a follow-up to a previous grant, please mention which one in the first line of the abstract and include a link to previous pull requests if applicable.
 
-## Checklist
+### Application Checklist
 
-- [ ] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied, renamed ( `project_name.md`) and updated.
+- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied, renamed ( `project_name.md`) and updated.
```
Comment (mmagician) [inline .github/pull_request_template.md:9]: Shouldn't be pre-ticked, no?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,19 +1,15 @@
...
 - [ ] The total funding amount of the project is below USD $30k for first-time grant applications and $100k for follow-up ones.
 - [ ] A BTC or Ethereum (DAI/USDT) address for the payment of the milestones is provided inside the application.
-- [ ] I have read and acknowledged the [Terms and Conditions](https://github.com/w3f/Grants-Program/blob/master/docs/T&Cs.md).
+- [ ] I have read and acknowledged the [terms and conditions](https://github.com/w3f/Grants-Program/blob/master/docs/T&Cs.md).
 - [ ] The software delivered for this grant will be released under an open-source license specified in the application.
-- [ ] The initial PR contains only one commit (squash if needed before submitting your PR).
+- [ ] :warning: The initial PR contains only one commit (squash and force-push if needed).
```
Comment (mmagician) [inline .github/pull_request_template.md:14]: that's quite explicit

File: .github/pull_request_template.md (hunk 1 above)
Comment (alxs) [reply .github/pull_request_template.md:9]: Seems not everybody knows how to tick these boxes, this is meant as an example. Following https://github.com/w3f/Grant-Milestone-Delivery/blob/master/.github/pull_request_template.md

File: .github/pull_request_template.md (hunk 2 above)
Comment (alxs) [reply .github/pull_request_template.md:14]: Might get a few more teams to do it

File: .github/pull_request_template.md (hunk 1 above)
Comment (mmagician) [reply .github/pull_request_template.md:9]: How about:

```
- [x] I have ticked this box
```

**Pull request merged**